ClosureCompilerRecursive.java

```diff
@@ -0,0 +1,81 @@
+import java.io.IOException;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.ArrayList;
+
+import com.google.javascript.jscomp.CompilationLevel;
+import com.google.javascript.jscomp.Compiler;
+import com.google.javascript.jscomp.CompilerOptions;
+import com.google.javascript.jscomp.SourceFile;
+import com.google.javascript.jscomp.CommandLineRunner;
+import com.google.javascript.jscomp.CompilerOptions.LanguageMode;
+
+public class ClosureCompilerRecursive {
+   protected Integer errorCount = 0;
+
+   public String compile(String filename, CompilerOptions options) {
+      Compiler compiler = new Compiler();
+
+      List<SourceFile> list = null;
```

Comment:
I think the `null` is implicit in declaring the variable, but I'm not sure what the Java convention is.

Reply:
If you don't, it complains if you try to reference it (`list may not initialized`)